src/components/subtitlesettings/subtitleappearancehelper.js

```diff
@@ -30,13 +30,13 @@ function getTextStyles(settings, preview) {
 
     switch (settings.dropShadow || '') {
         case 'raised':
-            list.push({ name: 'text-shadow', value: '-1px -1px white, 0px -1px white, -1px 0px white, 1px 1px black, 0px 1px black, 1px 0px black' });
+            list.push({ name: 'text-shadow', value: '-0.04em -0.04em #fff, 0px -0.04em #fff, -0.04em #fff, 0.04em 0.04em #000, 0px 0.04em #000, 0.04em 0px #000' });
             break;
         case 'depressed':
-            list.push({ name: 'text-shadow', value: '1px 1px white, 0px 1px white, 1px 0px white, -1px -1px black, 0px -1px black, -1px 0px black' });
+            list.push({ name: 'text-shadow', value: '0.04em 0.04em #fff, 0px 0.04em #fff, 0.04em #fff, -0.04em -0.04em #000, 0px -0.04em #000, -0.04em 0px #000' });
             break;
         case 'uniform':
-            list.push({ name: 'text-shadow', value: '-1px 0px #000000, 0px 1px #000000, 1px 0px #000000, 0px -1px #000000' });
+            list.push({ name: 'text-shadow', value: '#000 0px 0.04em, #000 0px -0.04em, #000 0px 0.07em, #000 0px -0.07em, #000 0.04em 0px, #000 -0.04em 0px, #000 0.04em 0.04em, #000 -0.04em 0.04em, #000 0.04em -0.04em, #000 -0.04em -0.04em, #000 0.04em 0.07em, #000 -0.04em 0.07em, #000 0.04em -0.07em, #000 -0.04em -0.07em, #000 0.07em 0px, #000 -0.07em 0px, #000 0.07em 0.04em, #000 -0.07em 0.04em, #000 0.07em -0.04em, #000 -0.07em -0.04em, #000 0.07em 0.07em, #000 -0.07em 0.07em, #000 0.07em -0.07em, #000 -0.07em -0.07em, #000 0.09em 0px, #000 -0.09em 0px, #000 0.09em 0.04em, #000 -0.09em 0.04em, #000 0.09em -0.04em, #000 -0.09em -0.04em' });
```

Comment:
This is the "circle" pattern I was talking about.
```suggestion
            list.push({ name: 'text-shadow', value: '#000 0px 0.03em, #000 0px -0.03em, #000 0px 0.06em, #000 0px -0.06em, #000 0.03em 0px, #000 -0.03em 0px, #000 0.03em 0.03em, #000 -0.03em 0.03em, #000 0.03em -0.03em, #000 -0.03em -0.03em, #000 0.03em 0.06em, #000 -0.03em 0.06em, #000 0.03em -0.06em, #000 -0.03em -0.06em, #000 0.06em 0px, #000 -0.06em 0px, #000 0.06em 0.03em, #000 -0.06em 0.03em, #000 0.06em -0.03em, #000 -0.06em -0.03em' });
```

Reply:
Ok added the circle pattern https://github.com/jellyfin/jellyfin-web/pull/3607/commits/8058f348b59efe145a177748543f194c77d01986